index.d.ts

```diff
@@ -153,9 +153,19 @@ import './src/foundry/collections/entityCollections/rollTables';
 import './src/foundry/collections/entityCollections/scenes';
 import './src/foundry/collections/entityCollections/users';
 
-import './src/foundry/diceTerms/coin';
-import './src/foundry/diceTerms/die';
-import './src/foundry/diceTerms/fateDie';
+import './src/foundry/rollTerm';
+
+import './src/foundry/rollTerms/diceTerm';
+import './src/foundry/rollTerms/mathTerm';
+import './src/foundry/rollTerms/numericTerm';
+import './src/foundry/rollTerms/operatorTerm';
+import './src/foundry/rollTerms/parentheticalTerm';
+import './src/foundry/rollTerms/poolTerm';
+import './src/foundry/rollTerms/stringTerm';
+
+import './src/foundry/rollTerms/diceTerms/coin';
+import './src/foundry/rollTerms/diceTerms/die';
+import './src/foundry/rollTerms/diceTerms/fateDie';
```

Comment:
You also still need to remove the old classes in the first section of the file.

Reply:
I moved the DicePool class into the PoolTerm file, that way it should be easier to find and remove later on